src/app/ui/editor/standby_state.cpp

```diff
@@ -325,6 +325,11 @@ bool StandbyState::onMouseDown(Editor* editor, MouseMessage* msg)
           1000, fmt::format("Layer '{}' is locked", layer->name()));
         return true;
       }
+      else if (!layer->isVisibleHierarchy()) {
+        StatusBar::instance()->showTip(
+          1000, fmt::format("Layer '{}' is hidden", layer->name()));
```

Comment:
I think we can move all these `"Layer '{}' is locked"` and `"Layer '{}' is hidden"` to `data/strings/en.ini` to avoid repeating. They could be in the `[statusbar_tips]` section.